scripts/end_to_end_test/smart_contract_tests/synergetic_mining.py

```diff
@@ -118,6 +118,8 @@ def run(options):
     print('Create contract...')
     api.sync(api.contracts.create(entity1, contract, 10000))
 
+    api.sync(api.tokens.transfer(entity1, contract.address, 10000, 500))
```

Comment:
Is this necessary for the test to pass?

Reply:
yes, otherwise no syn function will be executed